geth/wrapper.py

```diff
@@ -244,6 +245,9 @@ def construct_popen_command(data_dir=None,
     if allow_insecure_unlock:
         builder.append('--allow-insecure-unlock')
 
+    if cache:
+        builder.extend(("--cache", cache))
```

Comment:
total nit: double quotes where the rest of the file is using singles. (promise we'll implement black Soon™)

Reply:
haha! no prob